composer.json

```diff
@@ -24,7 +24,7 @@
         "illuminate/contracts": "5.1.* || 5.2.*",
         "illuminate/http": "5.1.* || 5.2.*",
         "illuminate/support": "5.1.* || 5.2.*",
-        "namshi/jose": "^6.0",
+        "namshi/jose": "^7.1",
```

Comment:
People might already use `7.0.0` in their project - forcing to use `^7.1` would cause package conflicts for them.